ov_msckf/src/slam2.cpp

```diff
@@ -272,7 +272,7 @@ class slam2 : public plugin {
 			));
 
 			_m_imu_integrator_input.put(_m_imu_integrator_input.allocate(
-				datum->time,
+				imu_cam_buffer->time,
```

Comment:
my guess is that this is wrong because imu_cam_buffer is only assigned to datum at the end of this function, so this is essentially using the next buffer's timestamp instead?

Reply:
Becasue the imu_cam_buffer is only assigned at the end of the function, so in each iteration we are using the time of the last daum. This is consistent with open_vins internally buffering the image for one iteration.